docs/cvl/invariants.md

```diff
@@ -163,13 +138,25 @@ block, if any), inside a set of curly braces (`{ ... }`).  Each preserved block
 consists of the keyword `preserved` followed by an optional method signature, 
 an optional `with` declaration, and finally the block of commands to execute.
 
+```{note}
+Although invariants are now checked on methods of all contracts on the scene,
+the method signature in a `preserved` block only allows specifying the method
+signature of the method, and not the receiver contract.  If multiple contracts
+on the scene implement methods with the same signature, you must use the same
+`preserved` block for all of them.
```

Comment:
Let's make a ticket to ensure we allow this flexibility in the future

Reply:
CERT-4003